src/common/transformations/tests/common_optimizations/group_normalization_fusion_tests.cpp

```diff
@@ -0,0 +1,570 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "common_test_utils/data_utils.hpp"
+#include "common_test_utils/ov_test_utils.hpp"
+#include "openvino/core/model.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/group_normalization.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/mvn.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/pass/manager.hpp"
+#include "openvino/pass/serialize.hpp"
+#include "transformations/common_optimizations/group_normalization_fusion.hpp"
+#include "transformations/init_node_info.hpp"
+
+using namespace testing;
+using namespace ov;
+
+template <element::Type_t T_act_elem,
+          element::Type_t T_gn_gamma_elem = T_act_elem,
+          element::Type_t T_gn_beta_elem = T_act_elem,
+          element::Type_t T_in_gamma_elem = T_act_elem,
+          element::Type_t T_in_beta_elem = T_act_elem>
+class GroupNormalizationFusionTestsFixture
+    : public ::testing::TestWithParam<
+          std::tuple<bool, PartialShape, Shape, Shape, Shape, Shape, unsigned long long, float>> {
+public:
+    static constexpr element::Type_t T_act_elem_t = T_act_elem;
+    static constexpr element::Type_t T_gn_gamma_elem_t = T_gn_gamma_elem;
+    static constexpr element::Type_t T_gn_beta_elem_t = T_gn_beta_elem;
+    static constexpr element::Type_t T_in_gamma_elem_t = T_in_gamma_elem;
+    static constexpr element::Type_t T_in_beta_elem_t = T_in_beta_elem;
+
+    typedef typename ov::element_type_traits<T_act_elem_t>::value_type T_act_store_t;
+    typedef typename ov::element_type_traits<T_gn_gamma_elem_t>::value_type T_gn_gamma_store_t;
+    typedef typename ov::element_type_traits<T_gn_beta_elem_t>::value_type T_gn_beta_store_t;
+    typedef typename ov::element_type_traits<T_in_gamma_elem_t>::value_type T_in_gamma_store_t;
+    typedef typename ov::element_type_traits<T_in_beta_elem_t>::value_type T_in_beta_store_t;
```

Comment:
```suggestion
    using DataType = ov::fundamental_type_for<ElementType>;
```
As all are same.
Maybe these types are not required at all (see. comment regarding test vector creation).

Reply:
Fixed in new version